.idea/workspace.xml

```diff
@@ -2,9 +2,17 @@
 <project version="4">
   <component name="ChangeListManager">
     <list default="true" id="68c36039-8adc-42d3-963c-c57117e2f839" name="Changes" comment="Fixed test for __str__ method for account type">
+      <change beforePath="$PROJECT_DIR$/.idea/LearnDjango.iml" beforeDir="false" afterPath="$PROJECT_DIR$/.idea/LearnDjango.iml" afterDir="false" />
```

Comment:
not sure if this file should be committed

Reply:
fixed